### ISSUE TITLE
Playback problem of ugoira converted to a video file with an uneven value as width/height.

### ISSUE THREAD

**Issue by diggafqsa** [Nandaka/PixivUtil2#1178]
### Prerequisites

* [x ] Did you read FAQ section in [readme.md](https://github.com/Nandaka/PixivUtil2/blob/master/readme.md)?
* [x ] Did you test with the latest [releases](https://github.com/Nandaka/PixivUtil2/releases "latest releases") or [commit](https://github.com/Nandaka/PixivUtil2/commits/master "latest commit") ?
* [x ] Did you search for existing issues in [Issues](https://github.com/Nandaka/PixivUtil2/issues)?

### Description

Trying to play video files converted from ugoiras that have an uneven value as height or width (ex. width:600 height:335) results in graphical artifacts during playback. 
And yes, I realize that this is not a problem with pixivutil per se, but there are ways to fix the uneven height/width values during ugoira-to-video conversion.

### Steps to Reproduce

1. Download and convert ugoira image id 92936913 (NSFW) to a video file.
2. Play it through a video player (MPC-HC x64 in my case).

**Expected behavior:**
No graphical artifacts during playback.
**Actual behavior:**
Has graphical artifacts during playback.
**Log file:** 
Log file not related to problem.

### Versions
Latest

### Possible fix to the problem
Inserting -vf pad='width=ceil(iw/2)*2:height=ceil(ih/2)*2' to ffmpegParam in config file.
This fixes the playback problem of video files that have an uneven value as height or width by padding the uneven height or width value with one line of black pixels. 
Solution copied from https://stackoverflow.com/questions/20847674/ffmpeg-libx264-height-not-divisible-by-2
Basically this takes the width and height of the input file and divides them by two, rounds them up to the closest integer using ffmpeg "ceil" expression and then multiplies by two. Will do nothing if height and width are already an even value.

**Comment by ManueLink64** [comment]
Try my personal configuration, I downloaded your ugoira file and works like a charm

```
[FFmpeg]
ffmpeg = ffmpeg.exe
ffmpegCodec = libvpx-vp9
ffmpegExt = webm
ffmpegParam = -row-mt 1 -deadline good -crf 19 -vsync 2 -r 999 -pix_fmt yuv444p
webpCodec = libwebp
webpParam = -row-mt 1 -lossless 0 -q:v 90 -loop 0 -vsync 2 -r 999
gifParam = -filter_complex "[0:v]split[a][b];[a]palettegen=stats_mode=diff[p];[b][p]paletteuse=dither=bayer:bayer_scale=5:diff_mode=rectangle"
apngParam = -vf "setpts=PTS-STARTPTS,hqdn3d=1.5:1.5:6:6" -plays 0
verboseOutput = True

[Ugoira]
writeUgoiraInfo = False
createUgoira = True
deleteZipFile = True
createGif = False
createApng = False
deleteUgoira = True
createWebm = True
createWebp = False
```

**Comment by diggafqsa** [comment]
Thanks for replying and sorry for late reply.

Looks like the culprit was the pixel formatting (-pix_fmt) set to yuv420p. Worked exactly as intended when changed to yuv444p though I does increase the file size quite a bit. I wonder if there is a way to force 444 formatting only when there are uneven values present in dimensions or if there are better ways of sorting this out without increasing filze sizes that much. Anyway, thanks for the help!
